indico/modules/legal/templates/tos_footer.html

```diff
@@ -1,5 +1,4 @@
-<br>
-<a href="{{ url_for('legal.display_tos') }}"
+| <a href="{{ url_for('legal.display_tos') }}"
```

Comment:
what about using `<li>` elements for footer text and adding the `|` via CSS?

Reply:
Fine for me. But how do we go about the extension point? Do we assume it returns items that are then separated with pipes?

Reply:
yeah, i don't think the ext point is good for anything but adding simple text items there anyway so expecting it to return `<li>..</li>` seems like the cleanest option